.github/workflows/fbgemmci.yml

```diff
@@ -248,6 +248,67 @@ jobs:
         python -c "import fbgemm_gpu"
         python -c "import fbgemm_gpu.split_embedding_codegen_lookup_invokers"
 
+  build_amd_gpu:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+
+    steps:
+    - name: Free space
+      run: sudo rm -rf /usr/local/android /usr/share/dotnet /usr/local/share/boost /opt/ghc /usr/local/share/chrom* /usr/share/swift /usr/local/julia* /usr/local/lib/android
```

Comment:
I don't think we need this step.

Reply:
Github actions VM is space-limited for ROCm and Pytorch. The CI throws a "No Space Left on Device" error when installing Pytorch.